specification/botservice/resource-manager/Microsoft.BotService/preview/2017-12-01/botservice.json

```diff
@@ -2340,6 +2147,26 @@
     }
   },
   "parameters": {
+    "resourceGroupNameParameter": {

```

Comment:
`resourceGroupNameParameter` and `resourceNameParameter` are missing the `x-ms-parameter-location` property, which can have some sorta gnarly code-gen repercussions.

I'd add `"x-ms-parameter-location":"method",` to both parameter definitions.